projects/hslayers-cesium/src/hover-popup.service.ts

```diff
@@ -0,0 +1,41 @@
+import {Injectable, NgZone} from '@angular/core';
+
+import {Feature} from 'ol';
+import {Geometry} from 'ol/geom';
+import {
+  HsConfig,
+  HsMapService,
+  HsQueryPopupBaseService,
+  HsQueryPopupServiceModel,
+  HsUtilsService,
+} from 'hslayers-ng';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HsCesiumQueryPopupService
```

Comment:
I would like to see the file named accordingly (query-popup.service.ts). Current name is misleading.

Reply:
Agree